<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLineDlg</class>
 <widget class="QDialog" name="EditLineDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Grueso</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="rdbH">
     <property name="text">
      <string>Horizontal</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="rdbV">
     <property name="text">
      <string>Vertical</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Estilo</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_3">
        <property name="text">
         <string>Solida</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButton_6">
        <property name="text">
         <string>Raya - punto</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_4">
        <property name="text">
         <string>Rayas</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_5">
        <property name="text">
         <string>Puntos</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="radioButton_7">
        <property name="text">
         <string>Raya - punto - punto</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btnCancelar">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnAceptar">
     <property name="text">
      <string>Aceptar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
